### PR TITLE
[pt-PT] Added rules to .AFFs taken from -BR to -PT 45+90

### PR DESCRIPTION
**Pull request by marcoagpinto**
Heya, @susanaboatto  and @p-goulart 

Here is an update of the pt-PT .AFFs “stolen” from pt-BR.

They added around 8 000 conjugations.

[6.PTPT_90_new_verbs.txt](https://github.com/languagetool-org/portuguese-pos-dict/files/14270938/6.PTPT_90_new_verbs.txt)

### CODE REVIEW

File: data/spelling-dict/hunspell/pt_PT_45.aff [hunk 1]
```diff
@@ -1270,7 +1270,283 @@ SFX K   0               mos             dizer           +P=1,N=p,T=ip
...
+SFX L   0       -lhe-�         ter
+SFX L   er      �-lo-�         ter
+SFX L   er      �-los-�        ter
+SFX L   0       -me-�          ter
+SFX L   0       -nos-�         ter
+SFX L   0       -te-�          ter
+SFX L   0       -vos-�         ter
+SFX L   0       -lhes-�o       ter
+SFX L   0       -me-�o         ter
+SFX L   0       -nas-�o        ter
```
Comment (p-goulart) [inline data/spelling-dict/hunspell/pt_PT_45.aff]: `ter-nas`?

File: data/spelling-dict/hunspell/pt_PT_45.aff [hunk 2]
```diff
@@ -1270,7 +1270,283 @@ SFX K   0               mos             dizer           +P=1,N=p,T=ip
...
+SFX L   r       nhamo-las      ter
+SFX L   r       nho-a          ter
+SFX L   r       nho-as         ter
+SFX L   r       nho-lhe        ter
+SFX L   r       nho-lhes       ter
+SFX L   r       nho-me         ter
+SFX L   r       nho-o          ter
+SFX L   r       nho-os         ter
+SFX L   r       nho-te         ter
+SFX L   r       ve-as          ter
```
Comment (p-goulart) [inline data/spelling-dict/hunspell/pt_PT_45.aff]: `teve-as` but no `teve-a`

File: data/spelling-dict/hunspell/pt_PT_45.aff [hunk 3]
```diff
@@ -1270,7 +1270,283 @@ SFX K   0               mos             dizer           +P=1,N=p,T=ip
...
+SFX L   r       ve-lhes        ter
+SFX L   r       ve-nos         ter
+SFX L   r       ve-o           ter
```
Comment (p-goulart) [inline data/spelling-dict/hunspell/pt_PT_45.aff]: `teve-o` but no `teve-os`

File: data/spelling-dict/hunspell/pt_PT_45.aff [hunk 4]
```diff
@@ -1270,7 +1270,283 @@ SFX K   0               mos             dizer           +P=1,N=p,T=ip
...
+SFX L   er      �m-lhe         ter
+SFX L   er      �m-lhes        ter
+SFX L   er      �m-me          ter
+SFX L   er      �m-na          ter
+SFX L   er      �m-nas         ter
+SFX L   er      �m-no          ter
+SFX L   er      �m-nos         ter
+SFX L   er      �m-se          ter
+SFX L   er      �m-te          ter
+SFX L   er      �m-vos         ter
```
Comment (p-goulart) [inline data/spelling-dict/hunspell/pt_PT_45.aff]: This outputs `tém-se`, etc. These should only be available for derived forms of `ter`, but not `ter` itself.

**Review by p-goulart — CHANGES_REQUESTED** [review]
This outputs a lot of verbs that we don't want, since the flag is not used only for verbs derived from `ter`. In the file attached to your PR we can see `acometer`, for example.

**Comment by marcoagpinto** [comment]
@p-goulart 

Ahhhh… thanks for the feedback.

Can the “ter” issues be solved if I add `[aeiou]ter` to `ter`?


Thanks!

**Comment by p-goulart** [comment]
> Can the “ter” issues be solved if I add `[aeiou]ter` to `ter`?

That won't account for all forms: `conter`, `obter`.

Trusting the Brazilian affix files too much is not wise since the flags are used in fundamentally different ways, with different distributions. It's best to just adapt the `pt-PT` flags.

**Comment by marcoagpinto** [comment]
Ahhhh… I will fix it tonight… or tomorrow at 5am… right now, I have been up since 3am working on other projects and my brain is drained, and I can't focus on precision things😄 😛 😛 😛 😛 😛 😛 😛 😛 😛 😛 😛

**Comment by marcoagpinto** [comment]
Heya, @p-goulart 

I have done some enhancements to the .AFFs.

Here are the results of the new AO90 one (new words):
[6.PTPT_90_new_verbs_20240216.txt](https://github.com/languagetool-org/portuguese-pos-dict/files/14303801/6.PTPT_90_new_verbs_20240216.txt)

I have been unable to check conjugation by conjugation, so, please, if you find any wrong item, please tell me with a suggestion on how to fix them.

Thanks!

**Comment by marcoagpinto** [comment]
@p-goulart 

Heya, Pedro, any news regarding this?

Thanks!

**Comment by p-goulart** [comment]
We can still spot some verb forms that don't make a lot of sense:
- abatém;
- batenha;
- combativesse;
- cometeve.

We know it's an unexciting task, but do you think you could check your work in full yourself and spot the odd ones out?

---

The approach you're taking here is not working because the flags are used differently in the `pt-PT` and `pt-BR` files. Please bear that in mind.

**Comment by marcoagpinto** [comment]
> We can still spot some verb forms that don't make a lot of sense:
> 
>     * abatém;
> 
>     * batenha;
> 
>     * combativesse;
> 
>     * cometeve.
> 
> 
> We know it's an unexciting task, but do you think you could check your work in full yourself and spot the odd ones out?
> 
> The approach you're taking here is not working because the flags are used differently in the `pt-PT` and `pt-BR` files. Please bear that in mind.

Ahhhh… thanks for the feedback, I will close this pull request and start with smaller rules… I was too greedy and wanted the bullet that cures all, or whatever it is called.

I always liked to make things bit by bit, but I was too greedy.

😋 😋 😋 😋 😋 😋 😋 😋 😋 😋 😋

**Pull request closed without merging**